subjects/Microchip/ASM/DioLAN/bootloader.dis

```diff
@@ -0,0 +1,168 @@
+// fn00000000
```

Comment:
`bootloader.asm` and `bootloader.dis` may have been added prematurely.

Reply:
Argh! Recommitted.